infra/base-images/base-builder/Dockerfile

```diff
@@ -23,7 +23,6 @@ RUN dpkg --add-architecture i386 && \
     apt-get update && \
     apt-get install -y \
         binutils-dev \
-        build-essential \
```

Comment:
It's possible that removing this will break some projects that now depend on build-essential being installed already.

Reply:
It is installed in base-clang, so base-builder inherits from it.

Reply:
OK. btw I'm not sure it is good that is installed btw. It may just be bloating our images without providing much benefit.

Reply:
Yes but i dont want to remove it now. also build-essential is needed for basic things, so good to keep it.